less/mixing.less

```diff
@@ -0,0 +1,6 @@
+.box-center(@justify: center, @items: center) {
+    display: flex;
```

Comment:
Nice use of mixins. I haven't used LESS in a while, so it's cool seeing all the stuff you're doing with it! :fire: